Rakefile

```diff
@@ -15,6 +22,10 @@ load "#{mruby_root}/Rakefile"
 
 desc "compile all the binaries"
 task :compile => [:mruby, :all] do
+  MRuby.each_target do |target|
+    `#{target.cc.command} --version`
+    abort("Command #{target.cc.command} for #{target.name} is missing.") unless $?.success?
+  end
```

Comment:
I guess into the rakefile method? could you explain me why do you output again those files via `setup.rb`?